client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.controller.js

```diff
@@ -104,6 +104,55 @@ angular.module("managerApp").controller("TelecomTelephonyServiceContactCtrl", fu
         });
     };
 
+    self.getFilteredDirectoryForm = function () {
+        OvhApiTelephony.Lexi().schema({
+            billingAccount: $stateParams.billingAccount,
+            serviceName: $stateParams.serviceName
+        }, {}).$promise.then(function (infos) {
+            const properties = infos.models && infos.models && infos.models["telephony.DirectoryInfo"] && infos.models["telephony.DirectoryInfo"].properties;
+            self.filteredDirectoryForm = _.pick(self.directoryForm, (v, k) => _.get(properties, [k, "readOnly"]) === 0);
```

Comment:
can we avoid (for the moment) to have a mix of old js and es6 ? Not that I don't like es6 but for a better readability, we should use the same anotations.